cspell.json

```diff
@@ -318,6 +318,7 @@
     "baseQueryResultHandler",
     "baseRefreshIntervalInSeconds",
     "bbox",
+    "bcfa",
```

Comment:
Added these words because they came up during `yarn spellcheck` as unknown words. These words were being used as IDs and so I added them in here to make the spellcheck happy